sympy/solvers/bivariate.py

```diff
@@ -168,17 +168,16 @@ def _lambert(eq, x):
 
     # solution of the given Lambert equation is like
     # sol = -c/b + (a/d)*LambertW(arg, k),
-    # where arg = d/(a*b)*exp((c*d-b*f)/a/b) and k in lambert_real_branches
-    # arg can have more than one values as exp(num/p*den)[exp((c*d-b*f)/a/b)]
+    # where arg = d/(a*b)*exp((c*d-b*f)/a/b) and k in lambert_real_branches.
+    # arg can have more than one values as exp(num/den)**(1/p)[exp((c*d-b*f)/a/b)]
```

Comment:
What does `exp(num/den)**(1/p)[exp((c*d-b*f)/a/b)]` mean?

Reply:
exp((c*d-b*f)/a/b) represented as exp(num/den)**(1/p). I will make it more clear.